scilpy/io/streamlines.py

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+from __future__ import division
 import os
+import six
 
 import nibabel as nib
+from nibabel.streamlines import Field, Tractogram
```

Comment:
to remove, use dipy.io.streamlines - load_tractogram and save_tractogram instead